frontend/src/layout/navigation/ProjectSwitcher.tsx

```diff
@@ -8,10 +8,13 @@ import { organizationLogic } from 'scenes/organizationLogic'
 import { sceneLogic } from 'scenes/sceneLogic'
 import { teamLogic } from 'scenes/teamLogic'
 import { urls } from 'scenes/urls'
-import { userLogic } from 'scenes/userLogic'
 import { AvailableFeature, TeamBasicType } from '~/types'
 import { navigationLogic } from './navigationLogic'
 
+type OnCloseProps = {
```

Comment:
We generally use a convention where `FooProps` means props of component `Foo`, and there is no `OnClose` component, so this break that. Maybe this'd be like `OnClosePropsMixin`. But probably the clearest way is just naming it `ProjectSwitcherOverlayProps`.